spec/data/node_spec.rb

```diff
@@ -331,6 +331,21 @@ def canonicalize(hw_info)
       node.installed.should == 'test'
     end
 
+    it "should update tags if the node is marked installed" do
+      node.installed = 'test'
+      node.save
+
+      policy = Fabricate(:policy, :rule_number => 20)
```

Comment:
Testing this shouldn't require a policy at all. Tags are matched regardless of which policies exist.